src/providers/completer/command.ts

```diff
@@ -248,14 +243,16 @@ export class Command implements IProvider {
                 }
                 result[1].split(',').forEach(pkg => {
                     pkg = pkg.trim()
-                    if (pkgs.includes(pkg)) {
+                    if (pkg === '') {
                         return
                     }
-                    const filePkgs = this.extension.manager.cachedContent[file].element.package
-                    if (filePkgs) {
+                    let filePkgs = this.extension.manager.cachedContent[file].element.package
+                    if (!filePkgs) {
+                        filePkgs = []
+                        this.extension.manager.cachedContent[file].element.package = filePkgs
+                    }
+                    if (filePkgs && !filePkgs.includes(pkg)) {
```

Comment:
It is correct what DeepScan is saying. We can change `filePkgs && !filePkgs.includes(pkg)` to `!filePkgs.includes(pkg)`.

Reply:
Great!